EasyVCR.Tests/CensorsTest.cs

```diff
@@ -0,0 +1,165 @@
+using System.Collections.Generic;
+using Microsoft.VisualStudio.TestTools.UnitTesting;
+
+namespace EasyVCR.Tests
+{
+    [TestClass]
+    public class CensorsTest
+    {
+        [TestMethod]
+        public void TestDefaultCensors()
+        {
+            var censors = Censors.Default;
+
+            Assert.IsTrue(true);
```

Comment:
question: what does this test actually do? We aren't making any valid assertions here